frontend/src/pages/notebookController/screens/server/SpawnerPage.tsx

```diff
@@ -85,6 +93,12 @@ const SpawnerPage: React.FC = () => {
   const [variableRows, setVariableRows] = React.useState<VariableRow[]>([]);
   const [createInProgress, setCreateInProgress] = React.useState<boolean>(false);
   const [submitError, setSubmitError] = React.useState<Error | null>(null);
+  const [isUsingCurrentTabFromStorage, setUsingCurrentTab] = useLocalStorage(
+    CURRENT_BROWSER_TAB_PREFERENCE,
+  );
+  const isUsingCurrentTab = isUsingCurrentTabFromStorage
+    ? JSON.parse(isUsingCurrentTabFromStorage)
+    : false;
```

Comment:
Move items into the component as per last comment. Also make this a hook by itself so you can use it in two spots.